### PR TITLE
Heat test to build a heat stack

### PR DESCRIPTION
**Pull request by ramak-suse**
Configured to run existing Heat tests to run from github.

### CODE REVIEW

File: scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/heat.sh.j2 [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+WORK_HOME={{ ardana_qe_test_work_dir }}
+
+AUTO_ID="AUTO`date +'%y%m%d%H%M%S'`"
+
+source ~/service.osrc
+
+cd {{ ardana_qe_tests_dir }}/ardana-qa-tests/heat/new-tests/
+
+# Run the test
+python HeatCluster.py  > heat.log
```
Comment (aojea) [inline scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/heat.sh.j2]: Where is this `HeatCluster.py` script

File: scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/heat.sh.j2 [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+
+# Save subunit output
+#cp ~/iverify_test$AUTO_ID/subunit_output ${WORK_HOME}/iverify.subunit
+
+
+# Run cleanup
+
```
[the remark this answers is not in the document]
Comment (ramak-suse) [reply scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/heat.sh.j2]: But I would like to keep the stack on the system.

File: scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/heat.sh.j2 (hunk 1 above)
Comment (ramak-suse) [reply scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/heat.sh.j2]: {{ ardana_qe_tests_dir }}/ardana-qa-tests/heat/new-tests/  directory.
The above directory points to the ardana-qe-tests repo.

File: scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/heat.sh.j2 (hunk 2 above)
[the remark this answers is not in the document]
Comment (flaviodsr) [reply scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/heat.sh.j2]: Could you explain why you want to keep the stack?

File: scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/vars/heat.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+ardana_qe_test_get_failed_cmd: "grep -e '^Stack.*\\.\\.\\..*FAIL' {{ ardana_qe_test_log }} || echo 'None'"
+ardana_qe_test_get_results_cmds:
+  passed: "grep -e '^Stack.*\\.\\.\\..*PASS' {{ ardana_qe_test_log }} | wc -l"
+  failed: "grep -e '^Stack.*\\.\\.\\..*FAIL' {{ ardana_qe_test_log }} | wc -l"
+  ran: "1"
```
[the remark this answers is not in the document]
Comment (ramak-suse) [reply scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/vars/heat.yml]: Fixed.

File: scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/heat.sh.j2 (hunk 2 above)
[the remark this answers is not in the document]
Comment (ramak-suse) [reply scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/heat.sh.j2]: As part of cleanup after the test  the image and the stack are deleted. Code checked in.

**Comment by ramak-suse** [comment]
Mistakenly cloned the SUSE-Cloud repo instead of the forked one.

**Pull request closed without merging**